roles/libvirt_manager/tasks/create_cloud_init_iso.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Generate cloud-init iso
+  become: false
+  vars:
+    _images_dir: "{{ [ cifmw_libvirt_manager_basedir, 'workload', ] | ansible.builtin.path_join }}"
+  block:
+    - name: "Acquire authorized_keys from hypervisor"
+      ansible.builtin.slurp:
+        src: "{{ ansible_user_dir ~ '/.ssh/authorized_keys' }}"
+      register: _authorized_keys_b64
+
+    - name: Create a list of ssh pub keys from authorized_keys
+      ansible.builtin.set_fact:
+        _ssh_authorized_keys: |-
+          [
+          {% for ssh_key in (_authorized_keys_b64.content | b64decode).split('\n') | select() %}
+            "{{ ssh_key | trim }}",
+          {% endfor %}
+          ]
+
+    - name: "Generate instance random UUID"
+      ansible.builtin.set_fact:
+        _instance_uuid: "{{ 99999999 | random(seed=vm) | to_uuid | lower }}"
+
+    - name: "Define the default users for the vm"
+      when:
+        - cifmw_libvirt_manager_vm_users is not defined or
+          cifmw_libvirt_manager_vm_users | length == 0
+      ansible.builtin.set_fact:
+        cifmw_libvirt_manager_vm_users:
+          - name: root
+            ssh_authorized_keys: "{{ _ssh_authorized_keys }}"
+            lock_passwd: false
+            plain_text_passwd: "{{ item.value.password | default('fooBar') }}"
+          - name: "{{ item.value.user | default('zuul') }}"
+            sudo: ALL=(ALL) NOPASSWD:ALL
+            ssh_authorized_keys: "{{ _ssh_authorized_keys }}"
+            lock_passwd: false
+            plain_text_passwd: "{{ item.value.password | default('fooBar') }}"
+
+    - name: "Define the user data"
+      ansible.builtin.set_fact:
+        _user_data:
+          users: "{{ cifmw_libvirt_manager_vm_users }}"
+          hostname: "{{ vm }}"
+          create_hostname_file: true
+          growpart:
+            devices: [/]
+            ignore_growroot_disabled: true
+            mode: growpart
+          resize_rootfs: noblock
+
+    - name: "Call the config_drive role"
+      vars:
+        cifmw_config_drive_iso_image: "{{ [ _images_dir, vm ~ '.iso' ] | ansible.builtin.path_join }}"
+        cifmw_config_drive_uuid: "{{ _instance_uuid }}"
+        cifmw_config_drive_hostname: "{{ vm }}"
+        cifmw_config_drive_networkconfig: "{{ item.value.networkconfig | default(omit) }}"
```

Comment:
omit doesn't work in vars, it's redered as the __omit__<uuid> string. It only works in module args

Reply:
*sigh* (thanks for the hint)